app/views/shared/_history_notice.html.erb

```diff
@@ -1,5 +1,4 @@
-<% if content_item.historically_political? %>
-  <div class="history-notice">
-    <h2>This was published under the <%= content_item.publishing_government %></h2>
-  </div>
+<% if content_item.historically_political?
+    text = 'This was published under the ' + content_item.publishing_government %>
+  <%= render 'components/banner', text: text %>
```

Comment:
I think this would be a little cleaner with [ruby string interpolation](https://stackoverflow.com/questions/10076579/string-concatenation-vs-interpolation-in-ruby):

```erb
<%= render 'components/banner', text: "This was published under the #{content_item.publishing_government}" %>
```